src/main/java/HttpServer/MockClient.java

```diff
@@ -0,0 +1,26 @@
+package HttpServer;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class MockClient implements Writable {
+
+    public ArrayList<String> output;
+    public boolean gotWrites;
+
+    public MockClient() {
+        this.gotWrites = false;
```

Comment:
`gotWrites` is kind of a strange name. Maybe try something like `wasToldToWriteLine`, `writeLineReceived`, `didReceiveWriteLine`.

Reply:
:white_check_mark: